pkg/config/remote/api/http.go

```diff
@@ -104,7 +115,10 @@ func (c *HTTPClient) Fetch(ctx context.Context, request *pbgo.LatestConfigsReque
 	if err != nil {
 		return nil, fmt.Errorf("failed to create org data request: %w", err)
 	}
+
+	c.headerLock.RLock()
 	req.Header = c.header
```

Comment:
this is not threadsafe. UpdatePARJWT will update req.Header too. 

https://play.golang.com/p/TWmOBaijsbF

I added a commit to copy the contents instead